### ISSUE TITLE
Templates not working

### ISSUE THREAD

**Issue by iv00020** [mkdo/cpd#44]
Hi guys,

I followed the instructions on the YouTube tutorial but when I reached the template phase where I should edit the Default Template and then be able to set each new user with it, I can't see it under the Templates section, nor can I create a new one. I am running _MultiSite_ and all the other requirements are met. I have never come across this which makes me think the plugin is blocking the templates for some reason. Do you have any idea why would this happen?

**Comment by mwtsn** [comment]
Hi @iv00020,

My only thoughts are that your multisite setup is not configured correctly. Can you create other sites without creating templates?

**Comment by iv00020** [comment]
Hi Matt,

I've worked with WordPress before and didn't have any problems creating a website or creating new templates. I reinstalled MultiSites 3 times and started the project from scratch but it still wouldn't work. Can I do it without using MultiSites? Or you mean can I create other sites with the MultiSites installed cause if this is your question then yes, I can create other websites.

Regards,
Ivan
